src/main/java/org/spongepowered/common/service/pagination/SpongePaginationService.java

```diff
@@ -102,36 +102,14 @@ void registerCommandOnce() {
         if (this.commandRegistered.compareAndSet(false, true)) {
             Sponge.getGame().getCommandDispatcher().register(Sponge.getPlugin(), CommandSpec.builder()
                     .description(t("Helper command for paginations occurring"))
-                    .arguments(new ActivePaginationCommandElement(t("pagination-id")))
```

Comment:
This reduces usability for console and other environments where the clickable text is not available.